packages/synapse-react-client/src/components/SynapseTable/EntityIDColumnCopyIcon.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react'
 import { useQueryContext } from '../QueryContext/QueryContext'
 import { useSynapseContext } from '../../utils/context/SynapseContext'
-import { InteractiveCopyIdsIcon } from '../InteractiveCopyIdsIcon'
+import {
+  InteractiveCopyIdsIcon,
+  InteractiveCopyIdsIconProps,
+} from '../InteractiveCopyIdsIcon'
 import { displayToast } from '../ToastMessage/ToastMessage'
 import { QueryResultBundle, Row } from '@sage-bionetworks/synapse-types'
 import { SynapseSpinner } from '../LoadingScreen'
 import { SynapseConstants } from '../../utils'
 import { getFullQueryTableResults } from '../../synapse-client/SynapseClient'
 import { parseEntityIdAndVersionFromSqlStatement } from '../../utils/functions/SqlFunctions'
 
-const EntityIDColumnCopyIcon = () => {
+type EntityIDColumnCopyIconProps = Omit<InteractiveCopyIdsIconProps, 'onCopy'>
```

Comment:
Since that here, you could either use "InteractiveCopyIdsIcon" or "IconButtonProps", does it make sense to stick with "IconButtonProps"? or it doesn't really matter what props that you use?